test_data/conus_2yr/README.md

```diff
@@ -0,0 +1,29 @@
+## conus_2yr
+
+This dir contains the version controlled files for this test domain.
+
+The actual run directory is configured as shown:
```

Comment:
I didnt mention but this baroque configuration is actually motivated by having the output on a faster disk than where the repository lives.

Reply:
I added some words about that now